Apps/frontend/src/assets/mui_publish.svg

```diff
@@ -0,0 +1 @@
+<svg xmlns="http://www.w3.org/2000/svg" height="24px" viewBox="0 0 24 24" width="24px" fill="#000000"><path d="M0 0h24v24H0z" fill="none"/><path d="M5 4v2h14V4H5zm0 10h4v6h6v-6h4l-7-7-7 7z"/></svg>
```

Comment:
What's the difference between publish and save. Is publish supposed to load a preset and "publish" it to the control panel? If it does, why not call it load?

Reply:
icon name is publish friom mui